_posts/2019-06-27-list-strategy.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: posts
+title: "List based execution strategy"
+authors:
+  - ishikhman
+---
+**tl;dr**: changes are coming with bazel 0.27 release: bazel supports list based execution strategy; build will fail if action cannot be executed with provided list of strategies; bazel will *auto-detect* a suitable strategy if none provided.
+
+## How it was before bazel 0.27 release
+Bazel supports a bunch of flags to configure an execution strategy:
+
+- The user can set the execution strategy in general and for individual action mnemonics using the flags `--spawn_strategy=`, 
+`--strategy=Mnemonic=` and `--strategy_regexp=Regex=`. The flags take a single strategy name as a value.  
+Example: `--spawn_strategy=linux-sandbox`
+
+- Bazel has a hardcoded and undocumented list of mnemonics that are known to work well with certain strategies and uses these 
+unless the user overrides them *individually* using `--strategy=Mnemonic=` flags.  
+Example: Bazel will use some kind of sandboxing by default if its available on your system, otherwise non-sandboxed execution (e.g. on Windows). However, it will run Javac actions via the persistent worker strategy.
+
+- Each strategy has custom code to deal with the situation that it can't execute a given action.  
+Example: the remote strategy silently falls back to sandboxed execution if an action can't run remotely. 
+The sandbox strategy silently falls back to non-sandboxed local execution if an action doesn't support sandboxing. 
+There's no good way to configure this behavior.
+
+### Example #1
```

Comment:
There's only one example under "How it was before bazel 0.27 release". Remove "#1"?